global/index.d.ts

```diff
@@ -5,16 +5,6 @@
  */
 declare var BridgeTalk: any;
 
-/**
- * The Infinity global property is a predefined variable with the value for infinity.
- */
-declare var Infinity: number;
-
-/**
- * The NaN global property is a predefined variable with the value NaN (Not-a-Number), as specified by the IEEE-754 standard.
- */
-declare var NaN: number;
-
```

Comment:
why you have removed Infinity and NaN?

Reply:
I just checked and I found out I was using the `"lib": ["es5"]` option, so it must have collided with that and I have removed those as a result.

Good catch =)